packages/api/src/router/event-group.ts

```diff
@@ -68,12 +64,12 @@ export const eventGroupRouter = router({
     .mutation(async ({ ctx, input }) => {
       const accounts = await ctx.prisma.account.findMany({
         where: { userId: ctx.user.id },
-        include: { color: true },
+        include: { userPrimary: true },
       });
       const accountEmails = accounts
         .map((a) => a.email)
         .filter((e): e is string => Boolean(e));
-      const primaryAccount = accounts.find((a) => a.isPrimary);
+      const primaryAccount = accounts.find((a) => Boolean(a.userPrimary));
```

Comment:
```suggestion
      const primaryAccount = accounts.find((a) => !!a.userPrimary);
```

nit but double bang ftw